src/neural-network.js

```diff
@@ -366,6 +392,22 @@ export default class NeuralNetwork {
     return true;
   }
 
+
+  /**
+   *
+   * @param options
+   * @private
+   */
+  _validateTrainingOptions(options) {
+    Object.keys(options).forEach(key => {
+      if (NeuralNetwork.trainingValidations[key] && !NeuralNetwork.trainingValidations[key](options[key])) {
+        let message = `[${key}, ${options[key]}] is out of noraml range`;
+        if (this._isStrict) throw new Error (message);
+        else console.log(message);
```

Comment:
`console.error`? `console.warn`?

Reply:
Good idea, I'll change it to console.warn